circe-akka-serializer/src/main/scala/org/virtuslab/ash/circe/CirceAkkaSerializer.scala

```diff
@@ -19,14 +19,19 @@ import org.reflections8.Reflections
 abstract class CirceAkkaSerializer[Ser <: AnyRef: ClassTag](system: ExtendedActorSystem)
     extends SerializerWithStringManifest
     with AkkaCodecs {
-  private val log = Logging(system, getClass)
-  private val conf = system.settings.config.getConfig("org.virtuslab.ash")
-  private val isDebugEnabled = conf.getBoolean("verbose-debug-logging") && log.isDebugEnabled
-
   val codecs: Seq[(ru.TypeTag[_ <: Ser], (Encoder[_ <: Ser], Decoder[_ <: Ser]))]
   val manifestMigrations: Seq[(String, String)]
   val packagePrefix: String
 
+  private val assertionMessage = " must be declared as a def or a lazy val to work correctly"
+  assert(codecs != null, "codecs" + assertionMessage)
+  assert(manifestMigrations != null, "manifestMigrations" + assertionMessage)
+  assert(packagePrefix != null, "packagePrefix" + assertionMessage)
+
+  private val log = Logging(system, getClass)
+  private val conf = system.settings.config.getConfig("org.virtuslab.ash")
+  private val isDebugEnabled = conf.getBoolean("verbose-debug-logging") && log.isDebugEnabled
```

Comment:
This can't ever throw a `ConfigException.Missing` right? since the key is present in reference.conf

Reply:
Yes, `akka-jackson-serializer` does the same thing.